src/test/java/com/epam/validator/ValidatorTest.java

```diff
@@ -1,3 +1,54 @@
 package com.epam.validator;
 
-public class ValidatorTest { }
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.time.LocalDate;
+
+public class ValidatorTest {
+
+    @Test(expected = NullPointerException.class)
+    public void checkEmptyThrowException(){
+        Validator.checkEmpty("");
+    }
+
+    @Test(expected = Exception.class)
+    public void checkDateDifferentThrowException(){
+        Validator.checkDateDifferent(LocalDate.of(2020,12,5),LocalDate.of(2018,12,5));
```

Comment:
May be you exctract this dates to local vars for understanding what wrong with them

Reply:
And in another test too. Please do not use some constant without vars, 'cause it difficult what wrong with them.
And let's start to use @DisplayName("meaningful text about what should happend in this test") over test methods

Reply:
This annotation was represented in JUNIT 5, which we cant use in our project cause it coflict with Travis.

Reply:
Done